Tests/IntegrationTestProject/ProjectUnderTest/ProjectUnderTest/CSProj1.csproj

```diff
@@ -27,7 +28,7 @@
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <WarningLevel>0</WarningLevel>
```

Comment:
There are unit/integration tests that check that build properties are set as expected. This property should be checked there. The AppVeyor check is too late and too general.